ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -57,6 +57,7 @@
 
   static final short VERSION_0 = 0;
   static final short VERSION_1 = 1;
+  static final short CURRENT_VERSION = VERSION_1;
```

Comment:
I don't think this needs to be a config. Value is always fixed (the latest one).